redbadger/website-honestly
# Add links to bank case study

### Motivation

Add the following for the new bank case study:
- Slice on 'our-work' page
- Entry for 'what to read next' at the bottom of case study pages
- Replace Camden slice on 'what we do' page

### Screenshots:

#### Slice on 'our-work' page, at the top of the page
![screen shot 2018-01-15 at 10 59 28](https://user-images.githubusercontent.com/5112255/34941140-64e7d452-f9ea-11e7-93b5-0889845dea1e.png)

#### Entry for 'what to read next' at bottom of case study pages
![screen shot 2018-01-15 at 10 59 17](https://user-images.githubusercontent.com/5112255/34941141-64faffa0-f9ea-11e7-99f5-bfcad7edfe8f.png)

#### Slice on 'what we do' page, replacing Camden Market slice
![screen shot 2018-01-15 at 11 22 57](https://user-images.githubusercontent.com/5112255/34941139-64d4eafe-f9ea-11e7-9d34-39c6c04e9f2b.png)


### Test plan

- Check `our-work` page
- Check the 'What to read next' section at the bottom of any case study (but not on the bank case study page itself, you might have to reload several times since they are populated randomly)
- Check the `what-we-do` page

### Pre-merge checklist
